EdurecaInClassProject/src/app/busesData.json

```diff
@@ -0,0 +1,16 @@
+[
+    {"busNumber":1020,"coach_type":"AC","from":"Bangalore","to":"Hubli","time":"8:00 PM","fare":850,"seats":36},
+    {"busNumber":1111,"coach_type":"NON-AC","from":"Bangalore","to":"Hubli","time":"9:00 PM","fare":750,"seats":36},
+    {"busNumber":1234,"coach_type":"AC","from":"Hubli","to":"Bangalore","time":"8:00 PM","fare":850,"seats":36},
+    {"busNumber":1212,"coach_type":"NON-AC","from":"Hubli","to":"Bangalore","time":"9:00 PM","fare":750,"seats":36},
+    {"busNumber":1122,"coach_type":"NON-AC","from":"Bangalore","to":"Pune","time":"8:00 PM","fare":870,"seats":36},
+    {"busNumber":1423,"coach_type":"AC","from":"Bangalore","to":"Pune","time":"9:00 PM","fare":950,"seats":36},
+    {"busNumber":1092,"coach_type":"NON-AC","from":"Pune","to":"Bangalore","time":"8:00 PM","fare":850,"seats":36},
```

Comment:
This JSON file should not come under the app folder directly. It can be stored in the asset folder.

Reply:
okay, i will do that